src/fides/api/ops/schemas/connection_configuration/connection_secrets_email.py

```diff
@@ -1,18 +1,52 @@
-from typing import List, Optional
+import abc
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel, Extra, root_validator
 
 from fides.api.ops.schemas.base_class import NoValidationSchema
 from fides.api.ops.schemas.connection_configuration.connection_secrets import (
     ConnectionConfigSecretsSchema,
 )
 
 
-class EmailSchema(ConnectionConfigSecretsSchema):
-    """Schema to validate the secrets needed for the EmailConnector"""
+class EmailSchema(BaseModel, abc.ABC):
```

Comment:
Created new `EmailSchema` base to avoid the `url` param from `ConnectionConfigSecretsSchema`